apps/codeforafrica/src/components/ProjectDetails/ProjectDescription.js

```diff
@@ -19,15 +21,17 @@ const ProjectDescription = React.forwardRef(
         ref={ref}
       >
         <RichTypography variant="h5">{title}</RichTypography>
-        <RichTypography variant="body1" sx={{ typography: { md: "body2" } }}>
-          {children}
-        </RichTypography>
+        <RichText
+          variant="body1"
+          elements={description}
+          typographyProps={{ sx: { typography: { md: "body2" } } }}
```

Comment:
```suggestion
          elements={description}
          typographyProps={{ sx: { typography: { md: "body2" } } }}
```

`variant="body1"` should be in `typographyProps` but `variant="body1"` is the default so we can just remove it.